web/src/styles/_text.scss

```diff
@@ -1,57 +1,53 @@
 .heading {
-	@apply font-[400] text-secondary;
+	@apply font-semibold text-primary mb-[32px];
+}
+
+.h1 {
+	@extend .heading;
+
+	@apply text-xl;
 
 	&::before {
 		content: ">_ ";
```

Comment:
I'd create a separate class for that to be more flexible :)

Reply:
You mean a separate for the prefix ">_" and one for a heading without it?

Reply:
exactly